subworkflows/local/minimap2_subworkflow.nf

```diff
@@ -0,0 +1,30 @@
+//
+// Minimap2 subworkflow to index an assembly and map reads to the assembly
```

Comment:
Probably no need to make this a subworkflow functionally at this time, but it's cool

Reply:
yeah for things that go together like indexing a reference and then mapping a reference I like to put it in a dedicated subworkflow just so that it's more clear. It follows how I've done the mapping_depth subworkflow for the illumina workflow and this minimap2 subworkflow will also have an added depth calculation step next, so this structure was to keep them consistent